Example/src/LightboxExample.tsx

```diff
@@ -97,14 +133,20 @@ function ListItem({ item, index, onPress }) {
       </Animated.View>
     </TapGestureHandler>
   );
-}
+};
 
 const timingConfig = {
   duration: 240,
   easing: Easing.bezier(0.33, 0.01, 0, 1),
 };
 
-function ImageTransition({ activeImage, onClose }) {
+function ImageTransition({
```

Comment:
Inconsistency with `const Name: FC<...>` syntax used throughout examples

Reply:
Fixed in 5ba3ad1.